src/ps-internal.h

```diff
@@ -59,7 +61,8 @@ typedef struct {
 
 /* Internal utilities */
 
-SEXP psll__is_running(ps_handle_t *handle);
+int psll__is_running(ps_handle_t *handle);
```

Comment:
This is a C variant of the `ps_is_running()` function that is used to compare creation times to avoid sending signals in case of reused PID.